src/GovUk.Education.ExploreEducationStatistics.Admin/Requests/FeaturedTableRequests.cs

```diff
@@ -7,9 +7,11 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Requests;
 public record FeaturedTableCreateRequest
 {
     [Required]
+    [MaxLength(120, ErrorMessage = "Featured table name must be 120 characters or less")]
```

Comment:
I'm wondering why you didn't add a FluentValidation Validator config to `FeaturedTableCreateRequest`, `FeaturedTableUpdateRequest`, and also `DataGuidanceUpdateRequest` and add this new validation in a consistent way?

I see two new FluentValidation classes being add in `ReleaseFileRequests.cs`, but not here.

From reviewing the changes it appears there's an inconsistent approach with how this new validation is being added.

Reply:
I think I was just sticking with whatever validation method was used in each class, I'll update those ones to use FV.